main/ejml-simple/src/org/ejml/simple/ops/SimpleOperations_DSCC.java

```diff
@@ -40,6 +43,10 @@
     public transient IGrowArray gw = new IGrowArray();
     public transient DGrowArray gx = new DGrowArray();
 
+    // Workspace for concurrent algorithms
+    public transient GrowArray<Workspace_MT_DSCC> workspaceMT = new GrowArray<>(Workspace_MT_DSCC::new);
+    public transient GrowArray<DGrowArray> workspaceA = new GrowArray<>(DGrowArray::new);
```

Comment:
What is the advantage of the `transient` keyword here. I read, its only useful for serialization.

Also do we assume users of SimpleOperations_DSCC wont several operations in parallel?
Parallel calls would be problematic when the same workspace is used

Reply:
To be honest I can't remember the exact reason it was used here since that code was written so long ago and there was no comment explaining why.

I had the same thought about parallel operation, but it already had workspace variables that would screw it up. No one has complained yet, but this should be thought through more carefully... Basically there will be no regression but the design might have already been flawed.

Reply:
just double checked and each SimpleMatrix has its own instance. Not very memory efficient but there will be no threading issue.